.github/workflows/macos.yml

```diff
@@ -13,6 +13,17 @@ on:
     # https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks
   merge_group:
 
+env:
+  # GITHUB_PULL_REQUEST_URL are used for commenting test reports in Launchable Github App.
+  # https://github.com/launchableinc/cli/blob/v1.80.1/launchable/utils/link.py#L42
+  GITHUB_PULL_REQUEST_URL: ${{ github.event.pull_request.html_url }}
+  # The following envs are necessary in Launchable tokenless authentication.
+  # https://github.com/launchableinc/cli/blob/v1.80.1/launchable/utils/authentication.py#L20
+  LAUNCHABLE_ORGANIZATION: "ruby"
+  LAUNCHABLE_WORKSPACE: "ruby"
```

Comment:
Can we set the GitHub's variables as a default value for the organization name and workspace name to make the code reusable in other repositories such as ruby/debug and in forked repositories such as junaruga/ruby rather than hard-coding the values?

```
LAUNCHABLE_ORGANIZATION: ${{ github.repository_owner }}
LAUNCHABLE_WORKSPACE: ${{ github.event.repository.name }}
```

For the `github.repository_owner`, I checked [this page](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context). For the `github.event.repository.name`, I checked [this page](https://stackoverflow.com/questions/62803531/repository-name-as-a-github-action-environment-variable). But I didn't test the values by myself yet.

Reply:
Updated

Reply:
Thanks. I confirmed that the variable set the organization value as "ruby" and the workspace value as "ruby" on this pull-request event's GitHub Actions log at least.

https://github.com/ruby/ruby/actions/runs/7912934050/job/21599616384?pr=9777#step:10:77

```
+ launchable verify
Organization: 'ruby'
Workspace: 'ruby'
...
```